main.py

```diff
@@ -82,39 +89,81 @@ def __main__():
+    query_operation_parser = operation_subparser.add_parser(stix_transmission.QUERY, help="Executes a query on the data source")
     query_operation_parser.add_argument('query_string', help='native datasource query string')
-    results_operation_parser = operation_subparser.add_parser(stix_shifter.RESULTS, help="Fetches the results of the data source query")
+    results_operation_parser = operation_subparser.add_parser(stix_transmission.RESULTS, help="Fetches the results of the data source query")
     results_operation_parser.add_argument('search_id', help='uuid of executed query')
     results_operation_parser.add_argument('offset', help='offset of results')
     results_operation_parser.add_argument('length', help='length of results')
-    status_operation_parser = operation_subparser.add_parser(stix_shifter.STATUS, help="Gets the current status of the query")
+    status_operation_parser = operation_subparser.add_parser(stix_transmission.STATUS, help="Gets the current status of the query")
     status_operation_parser.add_argument('search_id', help='uuid of executed query')
-    operation_subparser.add_parser(stix_shifter.IS_ASYNC, help='Checks if the query operation is asynchronous')
+    operation_subparser.add_parser(stix_transmission.IS_ASYNC, help='Checks if the query operation is asynchronous')
 
     args = parent_parser.parse_args()
 
     if args.command is None:
         parent_parser.print_help(sys.stderr)
         sys.exit(1)
 
-    shifter = stix_shifter.StixShifter()
 
     if args.command == TRANSLATE:
         options = json.loads(args.options) if bool(args.options) else {}
         if args.stix_validator:
             options['stix_validator'] = args.stix_validator
         if args.data_mapper:
             options['data_mapper'] = args.data_mapper
-        result = shifter.translate(
+
+        translation = stix_translation.StixTranslation()
+        result = translation.translate(
             args.module, args.translate_type, args.data_source, args.data, options=options)
     elif args.command == TRANSMIT:
-        result = shifter.transmit(args)
+        result = transmit(args) # stix_transmission
 
     print(result)
     exit(0)
 
 
+def transmit(args):
+    """
+    Connects to datasource and executes a query, grabs status update or query results
+    :param args:
+    args: <module> '{"host": <host IP>, "port": <port>, "cert": <certificate>}', '{"auth": <authentication>}',
+    <
+        query <query string>,
+        status <search id>,
+        results <search id> <offset> <length>,
+        ping,
+        is_async
+    >
+    """
+    connection_dict = json.loads(args.connection)
+    configuration_dict = json.loads(args.configuration)
+    transmission = stix_transmission.StixTransmission(args.module, connection_dict, configuration_dict)
+
+    operation_command = args.operation_command
+    if operation_command == QUERY:
```

Comment:
can we make it like `if operation_command == stix_transmission.QUERY` as the variable is already declared in the transmission